Add notes to clarify tenant functionality (#245)
alexkappa/terraform-provider-auth0: website/docs/r/tenant.html.md

With this resource, you can manage Auth0 tenants, including setting logos and support contact information, setting error pages, and configuring default tenant behaviors.

> #### Notes:
> Auth0 does not currently support creating or destryoying tenants through their API. This resource can only manage an existing tenant created through the Auth0 dashboard. 

Reviewer: ```suggestion
~> **Note**: Auth0 does not currently support creating tenants through the Management API. Therefore this resource can only manage an existing tenant created through the Auth0 dashboard. 
```

Auth0 does not currently support adding/removing extensions on tenants through their API. The Auth0 dashboard must be used to add/remove extensions. 

> 

Reviewer: I'm not sure whether the tenant resource is the best place to communicate this message. I would rather leave this out.

Author: I wavered on this too. I'm not sure of the best place to put this either. Maybe at the provider level documentation?

I do think it is important information to pass along. Just today I have been working with the Auth0 GitHub deployments, and again thought it would be good to know I cannot add/configure this extension through the auth0 provider.

I leave it up to you as to whether this warrants a note somehwhere, or not.

Thank you so much for your contribution with this provider. I hope to some day be able to make a more meaningful contribution.

Reviewer: Thanks for the feedback. I'll give some thought on where it's best to mention it.
